pkg/types/library.go

```diff
@@ -1,5 +1,12 @@
 package types
 
+import "github.com/aquasecurity/trivy-db/pkg/db"
+
+type VersionInfo struct {
```

Comment:
If db isn't in a local file system, this shows like the following. Is it intended?
```
{"TrivyVersion":"dev","VulnerabilityDBVersion":{}}
```

Reply:
If we handle the error like this https://github.com/aquasecurity/trivy/pull/435/files#diff-15cf00305be231c6aa506ff45d63ba69R252-R255 then we would have something like: `unable to display current version: unexpected end of JSON input`

What do you prefer? I have no strong opinions.

Reply:
Ah, I meant `VulnerabilityDBVersion` is empty here. We should show a version of Trivy anyway even if DB is not there.

You tried not to show NextUpdate and UpdatedAt if they are zero value, but you didn't do anything for `VulnerabilityDBVersion`. I just imagined you wanted to display something like this 😄 
```
{"TrivyVersion":"dev"}
```

Never mind. It doesn't matter even if empty `VulnerabilityDBVersion` is displayed.